Makefile.core.mk

```diff
@@ -475,4 +475,12 @@ include tools/packaging/packaging.mk
 #-----------------------------------------------------------------------------
 include tests/integration/tests.mk
 
+#-----------------------------------------------------------------------------
+# Target: bookinfo sample
+#-----------------------------------------------------------------------------
+.PHONY: bookinfo.build
+
+bookinfo.build:
+	@samples/bookinfo/src/build-services.sh ${VERSION} ${HUB}
```

Comment:
I think the answer is "no" but just double checking - this will NOT try to push the images?

Reply:
correct. Push is another shell script, that we might use in a future post-submit job.

Reply:
I'm not sure if there would be confusion, but VERSION here is the Istio version and not the current or next possible version of the Bookinfo images.

Reply:
@ericvn  Do you think we need another variable to hold the bookinfo version?

Reply:
I think that might be beneficial, especially if we are thinking about doing something with automation of building new images. Maybe we could read the current version from a yaml and just increment it as a value. I did think about changing the script to allow defaults for the Version (increment the current) and Hub (use the HUB env var). That would change its behavior but since it's really not used much, and not in automation, we could change that as needed.

Reply:
@ericvn done